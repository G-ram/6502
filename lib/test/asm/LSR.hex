a9 ff 8d 00 01 4e 00 01
