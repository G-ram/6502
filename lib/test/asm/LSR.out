A: FF
X: 00
Y: 00
P: 10010001
S: 20
PC: 608
100 : FF
101 : 00
